.vscode/launch.json

```diff
@@ -118,26 +108,45 @@
                 // Create a temp profile that has no extensions / user settings.
                 // This allows us to only have the C# extension + the dotnet runtime installer extension dependency.
                 "--profile-temp",
-                "${workspaceRoot}/test/integrationTests/testAssets/slnWithCsproj",
+                "${workspaceRoot}/test/integrationTests/testAssets/slnWithCsproj/.vscode/omnisharp_slnWithCsproj.code-workspace",
                 "--extensionDevelopmentPath=${workspaceRoot}",
                 "--extensionTestsPath=${workspaceRoot}/out/test/integrationTests"
             ],
             "env": {
-                "CODE_WORKSPACE_ROOT": "${workspaceRoot}",
-                "CODE_TESTS_PATH": "${workspaceRoot}/out/test/integrationTests",
-                "CODE_TESTS_WORKSPACE": "${workspaceRoot}/test/integrationTests/testAssets/slnWithCsproj",
                 "CODE_EXTENSIONS_PATH": "${workspaceRoot}",
                 "OSVC_SUITE": "slnWithCsproj",
-                "OMNISHARP_ENGINE": "stdio",
             },
-            "stopOnEntry": false,
             "sourceMaps": true,
             "outFiles": [
                 "${workspaceRoot}/dist/*.js",
                 "${workspaceRoot}/out/test/**/*.js"
             ],
             "preLaunchTask": "buildDev"
         },
+        {
+            "name": "Blue: Launch slnWithCsproj Workspace Tests",
```

Comment:
lets keep 'blue' out of the code.  Good names could be 'LSP tools host', 'LSP', 'Roslyn Server', etc.  Same for the rest of the file / folder names (not commenting them individually)

Reply:
could also consider having the new tests just be 'integrationTests' and renaming the O# folder/tests to 'omnisharpIntegrationTests'.  That I think might be the most clear